test/menuFactory.spec.js

```diff
@@ -413,5 +413,30 @@ describe('menuFactory', () => {
       ReactDOM.render(<Menu isOpen />, container);
       expect(component.state.isOpen).to.be.true;
     });
+
+    it('will not change when isOpen prop was not changed', () => {
+      let container = document.createElement('div');
+      let ParentComponent = React.createClass({
+        getInitialState() {
+          return { collapsed: true };
+        },
+        changeProps() {
+          this.setState({
+            collapsed: !this.state.collapsed
+          });
+        },
+        render() {
+          return (
+            <Menu ref="menu" {...this.state} />
+          );
+        }
+      });
```

Comment:
There's no `collapsed` prop, so to use spread attributes it needs to be:
```javascript
let ParentComponent = React.createClass({
  getInitialState() {
    return { isOpen: false };
  },
  changeProps() {
    this.setState({
      isOpen: !this.state.isOpen
    });
  },
  render() {
    return (
      <Menu ref="menu" {...this.state} />
    );
  }
});
```
or just:
```javascript
<Menu ref="menu" isOpen={ !this.state.collapsed } />
```
:)